csi/shell/entrypoint.sh

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 set -xe
 
+# if KUBELET_ROOTDIR not contains config.yaml, it is not likely a real kubelet rootdir, exit with 1
+if [ ! "$(ls $KUBELET_ROOTDIR/config.yaml)" ] ; then
+  echo "KUBELET_ROOTDIR [$KUBELET_ROOTDIR] is not likely a real kubelet rootdir, please configure csi.kubelet.rootDir in helm charts! "
+  exit 1
```

Comment:
Tthis logic is for a generic kubelet check, it is recommended to remove the helm charts related information or change it to a generic configuration option name.

For example: Error: $KUBELET_ROOTDIR does not contain config.yaml file, it's not a kubelet rootdir.

Reply:
I don't think check `config.yaml` is a good idea. It is possible that users put config.yaml elsewhere in customized K8s env. BTW, I suggest use `[ -d <dir> ]` or `[ -f <file> ]` instead of checking `ls <file or dir>`'s exit code.